### PR TITLE
fix(components/forms): updated single file attachment to use more descriptive aria labels

### PR DESCRIPTION
**Pull request by Blackbaud-TrevorBurch**

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1630/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/errors-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/forms-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/lookup-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/phone-field-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/1630/storybooks/theme-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1630/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1630/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1630/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1630/playground/)

File: apps/playground/src/app/components/forms/single-file-attachment/single-file-attachment.component.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
       (fileClick)="fileClick($event)"
     >
       <sky-file-attachment-label>
-        Attach any file
+        Birth certificate
```
Comment (Blackbaud-ErikaMcVey) [inline apps/playground/src/app/components/forms/single-file-attachment/single-file-attachment.component.html:11]: can we update the code example too? Attach any file doesn't make much sense with the new labels

File: libs/components/forms/src/assets/locales/resources_en_US.json [hunk 2]
```diff
@@ -11,14 +11,26 @@
...
+    "_description": "Button aria label for single file attachment when file has been chosen",
+    "message": "Replace file {0} for"
+  },
   "skyux_file_attachment_file_item_delete": {
     "_description": "Label for the button that deletes a file",
     "message": "Delete file"
   },
+  "skyux_file_attachment_file_item_remove": {
+    "_description": "Label for the button that deletes a file in single file attachment",
+    "message": "Remove {0} for"
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/forms/src/assets/locales/resources_en_US.json]: should be `"Remove file {0} for"`

**Review by Blackbaud-ErikaMcVey — CHANGES_REQUESTED** [review]
tested it with voiceover on chrome and safari. It worked well but I did notice one small issue--if the file name is really long we truncate it, so then the screenreader only reads the truncated bit instead of the full file file name.

File: apps/playground/src/app/components/forms/single-file-attachment/single-file-attachment.component.html (hunk 1 above)
Comment (Blackbaud-TrevorBurch) [reply apps/playground/src/app/components/forms/single-file-attachment/single-file-attachment.component.html:11]: Good catch - updated.

File: libs/components/forms/src/assets/locales/resources_en_US.json (hunk 2 above)
Comment (Blackbaud-TrevorBurch) [reply libs/components/forms/src/assets/locales/resources_en_US.json]: Oops - fixed.

**Review by Blackbaud-ErikaMcVey — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick [failed](https://github.com/blackbaud/skyux/actions/runs/5979698197)! :x: Please resolve conflicts and create a new PR.

**Pull request merged**